src/pages/HomePage/components/Unique/Unique.tsx

```diff
@@ -0,0 +1,36 @@
+import './Unique.scss';
+import furniture from './furniture.png';
+import ellipsered from './Ellipse65.svg' ;
+import ellipsecyan from './Ellipse67.svg' ;
+import ellipse from './Ellipse.svg';
+export const Unique = () => {
+  return (
+    <div className="unique">
+      <div className="unique-main">
+        <img className="furniture" src={furniture} alt="" />
+        <div className="information">
+          <p className='unique-features'>Unique Features Of leatest & Trending Poducts</p>
+          <div className='for-margin'>
+            <img className='red' src={ellipsered} alt="" />
```

Comment:
also it might be just a div with size, border-radius: 50% and some background-color )))